SWIG/swap.i

```diff
@@ -109,23 +109,17 @@ class FixedVsFloatingSwap : public Swap {
 %shared_ptr(VanillaSwap)
 class VanillaSwap : public FixedVsFloatingSwap {
```

Comment:
Thanks!  We can also enable kwargs while we're at it.

Reply:
Done